include/pocl.h

```diff
@@ -47,8 +47,13 @@
  */
 #define POCL_ADDRESS_SPACE_PRIVATE 0
 #define POCL_ADDRESS_SPACE_GLOBAL 1
+#ifdef HSA_RUNTIME_IS_ROCM
```

Comment:
This looks creepy. pocl is a multi-device platform so it can have other than HSA devices at the same time.